<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectUserGroupDialog</class>
 <widget class="QDialog" name="selectUserGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select User</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select User for Restrictions</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Selects a user or a group of users for which you want to apply the restrictions.</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
